### PR TITLE
johnstill/CB-22: view ICD9 search results in browser

### PR DESCRIPTION
**Pull request by johnstill**
This is a smallish set of changes to be able to see the results of searches in the browser (as reflected in summary charts) - ICD9 codes (both parent and leaves) are supported, inclusion and exclusion work as expected.  Note Bene: This PR requires the patch at https://github.com/all-of-us/workbench/pull/148/commits/3728aebbdea35452e82d39db4d987177177fb4fd in order to function properly.

### CODE REVIEW

File: ui/src/app/views/cohort-builder/search/cohort-builder/cohort-builder.component.ts [hunk 1]
```diff
@@ -164,55 +163,61 @@ export class CohortBuilderComponent implements OnInit, OnDestroy {
...
-      } else if (values[2] === 'H') {
-        h++;
-      } else if (values[2] === 'N') {
-        o++;
-      } else {
-        u++;
+      switch (race) {
+          case '1': races.Caucasian++           ; break;
+          case '2': races['African American']++ ; break;
+          default : races.Unknown++;
```
Comment (blrubenstein) [inline ui/src/app/views/cohort-builder/search/cohort-builder/cohort-builder.component.ts]: Does this need the other race options included in the switch statement, or is it just a placeholder? Or is this based on the `raceData` below?

File: ui/src/app/views/cohort-builder/search/cohort-builder/cohort-builder.component.ts [hunk 2]
```diff
@@ -164,55 +163,61 @@ export class CohortBuilderComponent implements OnInit, OnDestroy {
...
-        male++;
-      } else if (values[1] === 'F') {
-        female++;
-      } else {
-        unknown++;
+      const [uid, gender, race] = subject.val.split(',');
+      switch (gender) {
+          case '1': genders.Male++    ; break;
+          case '2': genders.Female++  ; break;
+          default : genders.Unknown++;
```
Comment (blrubenstein) [inline ui/src/app/views/cohort-builder/search/cohort-builder/cohort-builder.component.ts]: Is the survey just asking for binary gender, or is there an `other` field? (This is also partially for my curiousity.)

File: ui/src/app/views/cohort-builder/search/cohort-builder/cohort-builder.component.ts (hunk 1 above)
Comment (danqrodney) [reply ui/src/app/views/cohort-builder/search/cohort-builder/cohort-builder.component.ts]: Our race codes are different than all of this (and it's possible in future these counts will all be calculated by BQ server-side). I'm assuming this is just a placeholder.

File: ui/src/app/views/cohort-builder/search/cohort-builder/cohort-builder.component.ts (hunk 2 above)
Comment (danqrodney) [reply ui/src/app/views/cohort-builder/search/cohort-builder/cohort-builder.component.ts]: Same as race below, our gender options differ from this. Fine for a placeholder.

File: ui/src/app/views/cohort-builder/search/cohort-builder/cohort-builder.component.ts (hunk 1 above)
Comment (johnstill) [reply ui/src/app/views/cohort-builder/search/cohort-builder/cohort-builder.component.ts]: @danqrodney that is correct, this is a placeholder.  For both gender and race (and ethnicity is its own category in Big Query) I'd like to hold off on a final version of the chart-data generation code until we know what exactly the encoding will be, whether or not we want to generate the numbers via SQL or in the front end, whether we're using 3 queries for counts or incrementally doing counts, etc etc.

@blrubenstein I don't actually know if the survey (or whatever the original data source is) has a field for Other / Unknown / I'd rather not say.  The data we have in BQ right now is pretty neatly partitioned between male and female.

Comment (blrubenstein) [reply ui/src/app/views/cohort-builder/search/cohort-builder/cohort-builder.component.ts]: Sounds good! I just wanted to check. I assumed that it was placeholder, but it seemed worth a check just to be sure!

**Pull request merged**